src/main/scala/play/ext/i18n/loaders/YamlFileLoader.scala

```diff
@@ -27,7 +23,7 @@ class YamlFileLoader extends MessagesLoader {
   override def apply(messageSource: MessageSource, messageSourceName: String): Either[ExceptionSource, Map[String, String]] =
     try {
       // YAML document parser
-      val yaml = new Yaml(new Constructor(), new Representer(), new DumperOptions(), new CustomResolver())
+      val yaml = new Yaml()
```

Comment:
The default `Yaml` constructor looks the most like what we're trying to do here, just fill in all the defaults.